ts/shielder-sdk-crypto-wasm/src/circuits/withdraw.ts

```diff
@@ -45,6 +47,38 @@ export class WithdrawCircuit
     );
   }
 
+  async pubInputs(values: WithdrawAdvice): Promise<WithdrawPubInputs> {
+    if (!this.wasmCircuit) {
+      throw new Error("Circuit not initialized");
+    }
+    const pubInputsBytes = this.wasmCircuit.pub_inputs(
+      values.id.bytes,
+      values.nonce.bytes,
+      values.nullifierOld.bytes,
+      values.trapdoorOld.bytes,
+      values.accountBalanceOld.bytes,
+      values.tokenAddress.bytes,
+      values.path,
+      values.value.bytes,
+      values.nullifierNew.bytes,
+      values.trapdoorNew.bytes,
+      values.commitment.bytes
+    );
+    const pubInputs = splitUint8(pubInputsBytes, 7).map(
+      (bytes) => new Scalar(bytes)
+    );
```

Comment:
I need guidance (I have some blackout) - `7` is passed as a `chunk_size`, but it seems to be actually number of chunks